shared/src/metabase/mbql/normalize.cljc

```diff
@@ -169,6 +169,13 @@
     [:temporal-extract (normalize-tokens field :ignore-path) (maybe-normalize-token unit) (maybe-normalize-token mode)]
     [:temporal-extract (normalize-tokens field :ignore-path) (maybe-normalize-token unit)]))
 
+(defmethod normalize-mbql-clause-tokens :datetime-diff
+  [[_ x y unit]]
+  [:datetime-diff
+   (normalize-tokens x :ignore-path)
+   (normalize-tokens y :ignore-path)
+   (maybe-normalize-token unit)])
```

Comment:
Did you add tests for this?

Reply:
Now I did https://github.com/metabase/metabase/pull/25722/commits/e389aaf3fbfb6d434fa5a9aa8df89c7eeaa4dcc0